### PR TITLE
New translations to support enhanced login security.

### PR DESCRIPTION
**Pull request by efalsken**

### CODE REVIEW

File: src/DDTech.Localization/Admin.es.resx [hunk 1]
```diff
@@ -274,7 +274,7 @@
     <value>Columnas</value>
   </data>
   <data name="Settings_Nav_MaterialRequirements" xml:space="preserve">
-    <value>Material Requirements Planning</value>
+    <value>Planificación de requerimientos de material</value>
```
Comment (tynorton) [inline src/DDTech.Localization/Admin.es.resx]: Was this intentional?

Comment (efalsken) [inline src/DDTech.Localization/Admin.es.resx]: It wasn't, but it matches the translation on the line below, so I let it stay.

File: src/DDTech.Localization/Workbench.es.resx [hunk 2]
```diff
@@ -298,7 +298,7 @@
     <value>Retraso</value>
   </data>
   <data name="LeadTime" xml:space="preserve">
-    <value>Lead Time</value>
+    <value>Plazo de ejecución</value>
```
Comment (tynorton) [inline src/DDTech.Localization/Workbench.es.resx]: Intentional?

File: src/DDTech.Localization/Export.es.resx [hunk 3]
```diff
@@ -160,6 +160,6 @@
     <value>Seleccionar archivos a exportar:</value>
   </data>
   <data name="SupplyOrder" xml:space="preserve">
-    <value>Supply Order</value>
+    <value>Orden de suministro</value>
```
Comment (tynorton) [inline src/DDTech.Localization/Export.es.resx]: Intentional?

**Comment by tynorton** [comment]
Seems a bunch of `es-ES` stuff got updated that looks unrelated. We're probably going to want to revert that.

File: src/DDTech.Localization/Export.es.resx [hunk 4]
```diff
@@ -121,13 +121,13 @@
     <value>Bill of Materials</value>
   </data>
   <data name="BufferProfile" xml:space="preserve">
-    <value>Buffer Profile</value>
+    <value>Perfil de búfer</value>
```
Comment (tynorton) [inline src/DDTech.Localization/Export.es.resx]: missed a few

File: src/DDTech.Localization/Export.es.resx [hunk 5]
```diff
@@ -121,13 +121,13 @@
...
   </data>
   <data name="DataExport" xml:space="preserve">
     <value>Exportar Datos</value>
   </data>
   <data name="DemandOrder" xml:space="preserve">
-    <value>Demand Order</value>
+    <value>Orden de la demanda</value>
```
Comment (tynorton) [inline src/DDTech.Localization/Export.es.resx]: missed a few

File: src/DDTech.Localization/Admin.resx [hunk 6]
```diff
@@ -303,9 +303,12 @@
   <data name="Settings_User_MinimumLength" xml:space="preserve">
     <value>Minimum Length?</value>
   </data>
-  <data name="Settings_User_PasswordComplexity" xml:space="preserve">
+    <data name="Settings_User_PasswordComplexity" xml:space="preserve">
```
Comment (tynorton) [inline src/DDTech.Localization/Admin.resx]: ¯\_(ツ)_/¯

**Pull request merged**